src/js/ui/simply-pebble.js

```diff
@@ -1113,6 +1132,35 @@ SimplyPebble.accelConfig = function(def) {
   SimplyPebble.sendPacket(AccelConfigPacket.prop(def));
 };
 
+SimplyPebble.voiceDictationSession = function(callback) {
+  // If there's a transcription in progress
+  if (SimplyPebble.dictationCallback) {
+    callback( { 'status': -1, 'transcription': null } );
+    return;
+  }
+
+  // Grab the current window to re-show once we're done
+  SimplyPebble.window = WindowStack.top();
+
+  // Set the callback and send the packet
+  SimplyPebble.dictationCallback = callback;
```

Comment:
Use an array of callbacks similar to `SimplyPebble.accelPeek`. Packets are guaranteed order, so you know the first callback is always the one to call when a Packet comes in from the watch.

You'll probably want to start more information than just the callback, such as the top window, so you would have something like:
```
dictationListenerInfos.push({ callback: callback, window: WindowStack.top() });
```